tests/installation/installation_mode.pm

```diff
@@ -16,8 +16,10 @@ sub run() {
     }
 
     if ( get_var("ADDONURL") || get_var("ADDONS") ) {
-        send_key "alt-c";    # Include Add-On Products
-        assert_screen "addonproduct-included", 10;
+        if (!check_var('ADDONS', 'smt')) {  # no include for SMT add-on bnc928895
```

Comment:
You are right, but right now SMT can't be combined with any other add-on.
I asked Marita if testing SMT update in this phase makes sense as we use same ISO for installation and update (update without update).
This is code for unique way of updating SMT :(